docs/management/managing-saved-objects.asciidoc

```diff
@@ -37,12 +37,15 @@ and select *Relationships*.
 [[managing-saved-objects-export-objects]]
 === Import and export
 
-Using the import and export commands, you can move objects between different
+Using the import and export actions, you can move objects between different
```

Comment:
When I think "commands", I think of a CLI. But we are talking about a GUI here. I think "actions" makes a bit more sense.